generators/script/templates/.travis.yml

```diff
@@ -3,4 +3,6 @@ node_js:
   - "0.11"
   - "0.10"
 notifications:
-  email: false
\ No newline at end of file
+  email: false
+before_script:
+  - npm install coffee-script grunt-cli
```

Comment:
Yes.

About 2 weeks ago I enabled Travis CI for all my scripts. On Node 0.11, I seemed to be getting errors related to Grunt not being found, and on 0.10, it was the: `Warning: Cannot find module 'coffee-script/register' Use --force to continue.` error. Installing both of them before running the tests seemed to resolve the problem for me.